pkg/bridge/ai/register/register.go

```diff
@@ -8,20 +7,27 @@ import (
 	"github.com/sashabaranov/go-openai"
 	"github.com/yomorun/yomo/ai"
 	"github.com/yomorun/yomo/core/metadata"
-	"github.com/yomorun/yomo/pkg/bridge/mcp"
 )
 
-func init() {
-	ai.SetRegister(&register{})
+// MCPToolStore defines the ability to add and remove tools.
+type MCPToolStore interface {
```

Comment:
This interface definition is not extensible and requires more parameters.

Reply:
This interface does not need to be extensible; it exists only to resolve an import cycle.